### DISCUSSION TITLE
[FEATURE]arm64小盒子sysstat日志快满了

### DISCUSSION THREAD

**Discussion by xdd-c** [1Panel-dev/1Panel#6478] [Q&A]
### 1Panel 版本

v1.10.15-lts

### 请描述您的需求或者改进建议

黑豹盒子x2，arm64机器，4G+64G配置。挂载的分区/var/log为安装1panel生成，最近占用越来越大，快满了，大文件都集中在/var/log/sysstat目录，尝试修改过/etc/sysstat/sysstat配置，但日志占用还在增长，手动删除31个sa文件重启机器又会出现。
<img width="618" alt="Snipaste_2024-09-12_16-27-50" src="https://github.com/user-attachments/assets/cb4ce2d8-1290-4a2b-aae8-b383e80992df">
<img width="692" alt="Snipaste_2024-09-12_16-28-36" src="https://github.com/user-attachments/assets/5a5c8fd6-0ace-4c13-8fce-98c752ac4c90">

### 请描述你建议的实现方案

1、是否可以对小内存主机优化此处问题。
2、当下我应该修改哪里配置，还是扩容分区大小呢，但我觉得扩容不算根本解决。

**Comment by wanghe-fit2cloud** [comment]
1Panel 涉及的数据都会保存在安装目录中，与 `/var/log` 无关，`sar` 文件也并非由 1Panel 生成的。

**Comment by 5kon** [comment]
老兄是刷了armbian吗，参考下这贴：
https://forum.yunohost.org/t/armbian-zram-var-log-full-and-email-notifications-every-15-minutes/26295
